apps/server/src/modules/room/api/room.uc.ts

```diff
@@ -114,6 +116,32 @@ export class RoomUc {
 		await this.roomMembershipService.deleteRoomMembership(roomId);
 	}
 
+	public async copyRoom(userId: EntityId, roomId: EntityId, schoolId: EntityId): Promise<CopyStatus> {
```

Comment:
Do we really need `schoolId` as a parameter here? I'd suppose this to be the `schoolId` of the user executing the use case (by `userId`).